crates/bevy_render/src/camera/camera.rs

```diff
@@ -43,14 +58,21 @@ impl Default for DepthCalculation {
 }
 
 impl Camera {
+    pub fn get_window(&self) -> Option<WindowId> {
```

Comment:
Good, I like how this design plays out.

Reply:
Thanks. I like to add methods when removing fields if I can to ease the transition.